src/refract/TypeQueryVisitor.h

```diff
@@ -90,11 +90,11 @@ namespace refract
             }
 
             TypeQueryVisitor tq;
-            tq.visit(*e);
+            Visit(tq, *e);
 
             E type;
             TypeQueryVisitor eq;
-            type.content(eq);
+            Visitby(eq, type);
```

Comment:
Yes, that is why, this should be `VisityBy(type, eq)`. `type` is element and `eq` is visitor here.